docs/ci/github.mdx

```diff
@@ -54,38 +56,7 @@ Currently `setup-shorebird` only supports the latest stable version of shorebird
 
 ## Authentication
```

Comment:
Can we include the Authentication heading as part of the reusable CIAuthentication component?

Reply:
That was my first approach, but when doing so, the section would not show in the index 🙃

Reply:
Ah that’s annoying haha